backend-shared/plannershared.cpp

```diff
@@ -170,3 +171,43 @@ void plannerShared::set_min_switch_duration(int value)
 	// NO conversion, this is done in the planner model.
 	DivePlannerPointsModel::instance()->setMinSwitchDuration(value);
 }
+
+double plannerShared::bottomsac()
+{
+	return qPrefDivePlanner::bottomsac() / 1000.0;
+}
+void plannerShared::set_bottomsac(double value)
+{
+	// NO conversion, this is done in the planner model.
+	DivePlannerPointsModel::instance()->setBottomSac(value);
+}
```

Comment:
same asymetric setup - only worse with the factor of 1000 being hidden somewhere I can't see reading the commit.